cartridges/int_algolia/cartridge/scripts/algolia/helper/modelHelper.js

```diff
@@ -4,6 +4,8 @@ const logger = require('*/cartridge/scripts/algolia/helper/jobHelper').getAlgoli
 
 const COLOR_ATTRIBUTE_ID = 'color';
 
+const IS_PWA = false; // You can set this to true if you are using PWA, this is used to determine if we should return the variantID that necessary for PWA
```

Comment:
```suggestion
const IS_PWA = false; // You can set this to true if you are using PWA, this is used to determine if we should return the colorCode that necessary for PWA
```
Actually we could also include the colorCode systematically so it works for both SFRA and PWA by default. As you prefer.